transforms/create-element-to-jsx.js

```diff
@@ -60,11 +60,20 @@ module.exports = function(file, api, options) {
     }
   };
 
+  const jsxIdentifierNameFor = node => {
```

Comment:
Doing that would make the recursion harder to accomplish. The recursion is needed to allow for nested MemberExpressions.